app/util/smtp.py

```diff
@@ -0,0 +1,39 @@
+import smtplib
+
+
+class SMTP():
+    def __init__(self, host, port, user, password):
+        self.conn = None
+        self.host = host
+        self.port = port
+        self.user = user
+        self.password = password
+
+    def open(self):
+        if self.conn:
+            return False
+
+        self.conn = smtplib.SMTP(self.host, self.port)
+        self.conn.ehlo()
+        self.conn.starttls()
+        self.conn.login(self.user, self.password)
+
+        return self.conn
+
+    def close(self):
+        if self.conn:
+            return self.conn.quit()
+        return None
+
+    def send_message(self, mail_from, mailto, message):
+        if not mail_from or not mailto or not message:
+            return False
+
+        new_conn = self.open()
```

Comment:
so you open a new connection, send, close the connection, what if an exception happens and the connection never closes?

Reply:
fixed, connection is closing in finally block now